test/benchmarks/benchmark_adler32_copy.cc

```
@@ -147,6 +147,10 @@ BENCHMARK_ADLER32_COPY(neon, adler32_neon, adler32_copy_neon, test_cpu_features.
 BENCHMARK_ADLER32_COPY(neon_dotprod, adler32_neon_dotprod, adler32_copy_neon_dotprod, test_cpu_features.arm.has_neon && test_cpu_features.arm.has_dotprod);
 #endif
 
+#ifdef MIPS_MSA
+BENCHMARK_ADLER32_COPY(msa, adler32_msa, adler32_copy_msa, test_cpu_features.mips.has_msa);
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If BENCHMARK_DEFINE_F is a macro then please configure it.

(unknownMacro)